library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -278,13 +278,13 @@ class GroupTest {
         val group = runBlocking {
             alixClient.conversations.newGroup(
                 listOf(
-                    bo.walletAddress,
+                    boClient.inboxId,
                 )
             )
```

Comment:
A new group should take a list of inboxIds now correct? Instead of addresses?

Reply:
Still takes addresses. We can give you a method that takes `inbox_id`s as well if you think it's useful. Something like `create_group_with_inboxes(inbox_ids)`